MonteCarloSimulationAnalysis/probabilityDistributionFunctions/generateRandomUniformDistribution.m

```diff
@@ -20,6 +20,9 @@
 %      distribution : Random vector of noSamples uniformly distributed
 %       
 %% Function main body
-distribution = 2*standardDeviation*rand(noSamples,1) + meanValue - standardDeviation;
+intervalSize = 0.5*standardDeviation*sqrt(12);
```

Comment:
What is the reason behind these changes here?

Reply:
There was an inconsisteny in the formulas. All other distribution functions are defined to take in the standard deviation and mean value. With random uniform distribution however we had a mistake in our AFEM project code. 
If you put in for example meanValue = 10 and standardDeviation = 1 you would get a uniformly distributed values between 9 and 11. This makes sense, but that does not give back standard deviation of 1 as in the other cases. With the intervalSize we fix that problem as the interval size is calculated via standard deviation